src/main/kotlin/com/intellij/ml/llm/template/utils/EFCandidateFactory.kt

```diff
@@ -108,19 +117,21 @@ class EFCandidateFactory {
     private fun getLeftmostPsiElement(lineNumber: Int, editor: Editor, file: PsiFile): PsiElement? {
         // get the PsiElement on the given lineNumber
         var psiElement = file.findElementAt(editor.document.getLineStartOffset(lineNumber)) ?: return null
+        var psiElementEndOffset = file.findElementAt(editor.document.getLineEndOffset(lineNumber)) ?: return null
 
         // if there are multiple sibling PsiElements on the same line, look for the first one
-        while (psiElement.getLineNumber(true) == psiElement.prevSibling?.getLineNumber(true)) {
+        while (psiElement.getLineNumber(false) == psiElement.prevSibling?.getLineNumber(false)) {
```

Comment:
True means it will return the line of the start offset, and False means it will return the line of the end offset. I changed it because I need the end offset now.